src/Controller/BackendController.php

```diff
@@ -210,4 +212,20 @@ public function alertsAction()
 
         return $controller->run();
     }
+
+    /**
+     * Handles the picker redirect.
+     *
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     *
+     * @Route("/_contao/picker", name="contao_backend_picker")
+     */
+    public function pickerAction(Request $request)
+    {
+        $pickerBuilder = $this->container->get('contao.menu.picker_menu_builder');
+
+        return new RedirectResponse($pickerBuilder->getPickerUrl($request->query->all()));
```

Comment:
I don't know if this is sufficient:

```php
public function getPickerUrl(Request $request)
{
    if ($request->query->has('value')) {
        foreach ($this->providers as $provider) {
            if ($provider->canHandle($request->query->get('value'))) {
                return $provider->getPickerUrl($request);
            }
        }
    }

    return $this->router->generate('contao_backend', array_merge(['do' => 'page'], $request->query->all()));
    }
```

The code still checks `$request->query->get('value')`. Is this ok?